src/Factory/Api/Twitter/Status.php

```diff
@@ -119,7 +119,21 @@ private function createFromArray(array $item, int $uid, bool $include_entities):
 
 		$friendica_comments = Post::countPosts(['thr-parent-id' => $item['uri-id'], 'deleted' => false, 'gravity' => GRAVITY_COMMENT]);
 
-		$text = Post\Media::addAttachmentsToBody($item['uri-id'], $item['body']);
+		$text = '';
+
+		// Add the title to text / html if set
+		if (!empty($item['title'])) {
+			$text .= $item['title'] . ' ';
+
+			$title         = sprintf("[h4]%s[/h4]\n", $item['title']);
+			$statusnetHtml = BBCode::convertForUriId($item['uri-id'], BBCode::setMentionsToNicknames($title . $item['raw-body'] ?? $item['body']), BBCode::API);
+			$friendicaHtml = BBCode::convertForUriId($item['uri-id'], $title . $item['body'], BBCode::EXTERNAL);
+		} else {
+			$statusnetHtml = BBCode::convertForUriId($item['uri-id'], BBCode::setMentionsToNicknames($item['raw-body'] ?? $item['body']), BBCode::API);
+			$friendicaHtml = BBCode::convertForUriId($item['uri-id'], $item['body'], BBCode::EXTERNAL);
+		}
```

Comment:
In `$title . $item['raw-body'] ?? $item['body']`, `$item['body']` will never be used because `$title` is defined the line before and the concatenation operator is evaluated before the null-coalescing `??` operator. I assume you wanted something like `$title . ($item['raw-body'] ?? $item['body'])` instead.

But then there's a simpler way to write all this:
```suggestion
		$title = '';
		$text = '';

		// Add the title to text / html if set
		if (!empty($item['title'])) {
			$text = $item['title'] . ' ';
			$title = sprintf("[h4]%s[/h4]\n", $item['title']);
		}

		$statusnetHtml = BBCode::convertForUriId($item['uri-id'], BBCode::setMentionsToNicknames($title . ($item['raw-body'] ?? $item['body'])), BBCode::API);
		$friendicaHtml = BBCode::convertForUriId($item['uri-id'], $title . $item['body'], BBCode::EXTERNAL);
```